### PR TITLE
chore , refactor : removed ! rm lightweight_pipeline.json , Added cleanup code for deletion of pipeline and refactored code according to template guidelines , performed linter test

### PR DESCRIPTION
**Pull request by Jayakrishna2801**
This PR includes the following updates:

Adds Colab Enterprise link and other structural changes from the new template.
Changes REGION variable name to LOCATION
Adds clean up code for pipeline deletion 
Removes IS_TESTING from the cleaning up section.
Removes the usage of future tense.

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

2. If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

<br>

3. If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/community-content) folder:
- [ ] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [ ] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/community-content/CODEOWNERS) file under the `Community Content` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

### CODE REVIEW

File: notebooks/official/pipelines/lightweight_functions_component_io_kfp.ipynb [hunk 1]
```diff
@@ -1,874 +1,771 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "pipeline_constants"
+   },
+   "source": [
+    "#### Vertex AI Pipelines constants\n",
+    "\n",
+    "Setup up the following constants for Vertex AI Pipelines:"
```
Comment (kittyabs) [inline notebooks/official/pipelines/lightweight_functions_component_io_kfp.ipynb:364]: Change: 

- Setup up the following constants for Vertex AI Pipelines

To:

- Set up the following constants for Vertex AI Pipelines

File: notebooks/official/pipelines/lightweight_functions_component_io_kfp.ipynb [hunk 2]
```diff
@@ -1,874 +1,771 @@
...
+    "\n",
+    "    with open(model.path, \"w\") as f:\n",
+    "        f.write(\"My Model\")\n",
+    "\n",
+    "    with open(imported_dataset.path, \"r\") as f:\n",
+    "        data = f.read()\n",
+    "    print(\"Imported Dataset:\", data)\n",
+    "\n",
+    "    # Use model.get() to get a Model artifact, which has a .metadata dictionary\n",
+    "    # to store arbitrary metadata for the output artifact. This metadata will be\n",
```
Comment (kittyabs) [inline notebooks/official/pipelines/lightweight_functions_component_io_kfp.ipynb:536]: change "will be" to "is"

File: notebooks/official/pipelines/lightweight_functions_component_io_kfp.ipynb [hunk 3]
```diff
@@ -1,874 +1,771 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "view_pipeline_run:lightweight"
+   },
+   "source": [
+    "Click on the generated link to see your run in the Cloud Console.\n",
+    "\n",
+    "<!-- It should look something like this as it is running:\n",
```
Comment (kittyabs) [inline notebooks/official/pipelines/lightweight_functions_component_io_kfp.ipynb:691]: change "it is" to "it's"

File: notebooks/official/pipelines/lightweight_functions_component_io_kfp.ipynb [hunk 4]
```diff
@@ -1,874 +1,771 @@
...
+    "\n",
+    "<a href=\"https://storage.googleapis.com/amy-jo/images/mp/automl_tabular_classif.png\" target=\"_blank\"><img src=\"https://storage.googleapis.com/amy-jo/images/mp/automl_tabular_classif.png\" width=\"40%\"/></a> -->\n",
+    "\n",
+    "In the UI, many of the pipeline DAG nodes will expand or collapse when you click on them. Here is a partially-expanded view of the DAG (click image to see larger version).\n",
```
Comment (kittyabs) [inline notebooks/official/pipelines/lightweight_functions_component_io_kfp.ipynb:695]: - Question, is "UI reference to the Google Cloud console? If so, change "UI" to "Google Cloud console.

- remove "will" from "will expand"

**Review by kittyabs — COMMENTED** [review]
suggested edits

**Review by katiemn — APPROVED** [review]

**Comment by katiemn** [comment]
Please take a look at the format and lint failures

**Pull request merged**